frontend/src/app/app.component.spec.ts

```diff
@@ -1,31 +1,25 @@
 import { TestBed, async } from '@angular/core/testing';
+
 import { AppComponent } from './app.component';
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-login-form',
+  template: '',
+})
+class StubLoginFormComponent {}
 
```

Comment:
Hmm, I'm concerned how many stubs we'll need for more complex components... I remember there was an option to force Angular to ignore template errors in tests.